base/changes.txt

```diff
@@ -6,6 +6,12 @@ to completeness or accuracy and it contains some references to files that are
 not part of the distribution.
 ================================================================================
 
+2025-03-01  Frank Mittelbach  <Frank.Mittelbach@latex-project.org>
+
+	* ltcounts.dtx (subsection{Environment Counter Macros}):
```

Comment:
This subsection title is odd. There is no "environment counter" now. Could be resolved in a separate PR.

https://github.com/latex3/latex2e/blob/4f1d77ce355109250987fcbe128a692d612ba834/base/ltcounts.dtx#L139-L173

Reply:
I changed things a bit, but these files really would need a more general docu update one day, by the look of it (as it is still a mix of old 2.09 docu with much of it no longer correct byx now)